backend/application/resources/auth.py

```diff
@@ -0,0 +1,40 @@
+from ...domain.models.auth import check_pw_pattern
+from ...app import app
+from apiflask import APIFlask, APIBlueprint, abort
+import bcrypt
+from infrastructure.database import db
+from infrastructure.database.entities import User
+
+router = APIBlueprint("authentications", __name__)
+
+
+@router.post("/SignIn")
+@input("name", str)
+@input("email", str)
+@input("password", str)
+def signin(name, email, password):
+    existing_user = User.query.filter_by(email=email).first()
+    if existing_user:
+        return {"message": "Email já cadastrado", "code": 400}
+    elif not (check_pw_pattern(password)):
+        return {"message": "Senha fraca", "code": 400}
+    else:
+        hashed_password = bcrypt.hashpw(password.encode("utf-8"), bcrypt.gensalt())
+        new_user = User(name=name, email=email, password=hashed_password)
+        db.session.add(new_user)
+        db.session.commit()
+        return {"message": "Usuário cadastrado com sucesso", "code": 201}
+
+
+@router.post("/Login")
```

Comment:
Escreva o path dos endpoints em minúsculo

```suggestion
@router.post("/signin")
```